src/main/resources/application.conf

```diff
@@ -101,7 +109,7 @@ grothendieck {
       max-mpt-components-per-message = 400
 
       # Maximum connected peers
-      max-peers = 10
+      max-peers = 20
```

Comment:
usually most of them are the ones that we keep trying to connect, not really connected peers. It seems to make sense to increase it to default 20 (maybe even 30)?